service/test/unit/config/test_leap.py

```diff
@@ -126,4 +127,53 @@ def test_should_add_welcome_email_on_a_fresh_account(self, mock_create_leap_sess
 
         self._user_bootstrap.setup(self.user_auth, self.password, '')
 
+        # TODO: this is incorrect
```

Comment:
Good point. The problem was that the code didn't actually assert anything - it used called_once_with when it should have used assert_called_once_with   -- I actually fixed this in another of my pull requests so I'm pushing a commit to remove the comment.